apps/studio/components/ui/BannerStack/BannerStack.tsx

```diff
@@ -0,0 +1,79 @@
+// apps/studio/components/ui/BannerStack/BannerStack.tsx
```

Comment:
Or (following up from my comment above) is this a pattern you think we should stick to? Reiterating the relative path at the top of the file.